README.md

```diff
@@ -32,6 +32,11 @@ Get the CLI binaries from our [Releases page](https://github.com/rainforestapp/r
 
 The CLI will check for updates and automatically update itself on every use unless the global flag `--skip-update` is used.
 
+You can download the latest (Linux) `rainforest-cli` binary with the following one-liner (requires `curl`, `jq` and `tar`):
```

Comment:
Sounds good!

If you have a concrete suggestion like this, then please free to use the "Insert a suggestion" Github feature - it cuts down on any ambiguity.

Normally, when it looks like the PR is mostly done and there's some minor feedback but it otherwise looks safe to ship we just approve it and leave comments. This assumes that the PR author isn't going to do the right thing, and means they are free to merge and don't have to wait for another review.

It's up the to the PR author to decide if they have reached the acceptable quality level or need some reassurance before shipping after an earlier review.

Reply:
@CChanHY can you review please!